infra/modules/instance-from-tpl/metadata.yaml

```diff
@@ -0,0 +1,285 @@
+apiVersion: blueprints.cloud.google.com/v1alpha1
```

Comment:
What are these metadata files? They look generated. Do we need them in our codebase?

Reply:
This is used as a template to send to google when creating an instance template, we do need this in the codebase